<!DOCTYPE html>
<html>
<head></head>
<body>
<h3>SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), version v1.6.6&nbsp;&nbsp;(<a href="https://github.com/jaleezyy/covid-19-signal">https://github.com/jaleezyy/covid-19-signal</a>)</h3>
<p>Note: Asterisks (*) indicates a discrepancy between iVar (default) and FreeBayes (if run)</p>
<h3>Sample: benchmark-bad16</h3>
<p><table><tr>
<td style="vertical-align: top"><table>
<tr><th colspan="2" style="text-align: left">Pangolin Lineage Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">A</td></tr>
<tr><th colspan="2" style="text-align: left">Nextclade Clade Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">19B</td></tr>
<tr><th colspan="2" style="text-align: left">Data Volume [ <a href="adapter_trimmed/benchmark-bad16_trim_galore.log">benchmark-bad16_trim_galore.log</a> ]</th>
<tr><td style="padding-left: 20px">Raw Data (read pairs)</td><td>314516</td></tr>
<tr><td style="padding-left: 20px">Raw Data (base pairs)</td><td>90993377</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (read pairs)</td><td>314516</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (base pairs)</td><td>90243124</td></tr>
<tr><td style="padding-left: 20px">Post Trim (read pairs)</td><td>314182</td></tr>
<tr><th colspan="2" style="text-align: left">Quality Control Flags</th>
<tr><td style="padding-left: 20px">Genome Fraction greater than 90%</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">No indels detected (maximum length 85bp)</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Depth of coverage >= 2000x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">All variants with at least 90% frequency among reads</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Frameshifts in SARS-CoV-2 open reading frames</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Reads per base sequence quality</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Sequencing adapter removed</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 100x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 1000x</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">FASTQC Flags [ <a href="adapter_trimmed/benchmark-bad16_R1_val_1_fastqc.html">benchmark-bad16_R1_val_1_fastqc.html</a> | <a href="adapter_trimmed/benchmark-bad16_R2_val_2_fastqc.html">benchmark-bad16_R2_val_2_fastqc.html</a> ]</th>
<tr><td style="padding-left: 20px">Per base sequence content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Sequence Length Distribution</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">Kraken2 [ <a href="kraken2/benchmark-bad16_kraken2.report">benchmark-bad16_kraken2.report</a> ]</th>
<tr><td style="padding-left: 20px">Reads SARS-CoV-2 (%)</td><td>92.62</td></tr>
<tr><th colspan="2" style="text-align: left">QUAST [ <a href="quast/benchmark-bad16_quast_report.html">benchmark-bad16_quast_report.html</a> ]</th>
<tr><td style="padding-left: 20px">Genome Length (bp)</td><td>29903.0</td></tr>
<tr><td style="padding-left: 20px">Genome Fraction (%)</td><td>98.515</td></tr>
<tr><td style="padding-left: 20px">N's per 100 kbp</td><td>1514.9</td></tr>
<tr><td style="padding-left: 20px">Genomic Features</td><td>19 + 5 part</td></tr>
<tr><td style="padding-left: 20px">Mismatches</td><td>4.0</td></tr>
<tr><td style="padding-left: 20px">Mismatches per 100 kbp</td><td>13.58</td></tr>
<tr><td style="padding-left: 20px">Indels</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">Indels per 100 kbp</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">Average Depth of Coverage</td><td>2873.8</td></tr>
<tr><td style="padding-left: 40px">Fraction with 0 coverage</td><td>0.014</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1x-10x coverage</td><td>0.001</td></tr>
<tr><td style="padding-left: 40px">Fraction with 11x-100x coverage</td><td>0.019</td></tr>
<tr><td style="padding-left: 40px">Fraction with 101x-1000x coverage</td><td>0.173</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1001x-2000x coverage</td><td>0.176</td></tr>
<tr><td style="padding-left: 40px">Fraction with 2001x-10000x coverage</td><td>0.604</td></tr>
<tr><td style="padding-left: 40px">Fraction with > 10000x coverage</td><td>0.013</td></tr>
<tr><td style="padding-left: 20px">5' Ns</td><td>45</td></tr>
<tr><td style="padding-left: 20px">3' Ns</td><td>67</td></tr>
<tr><th colspan="2" style="text-align: left">Variants in Consensus Genome (iVar)</th>
<tr><td colspan="2" style="padding-left: 20px">C3874T C8782T C12452-CT* C18060T G19565T A25175T T28144C</td></tr>
<tr><th colspan="2" style="text-align: left">Unique Variants in Consensus Genome (FreeBayes)</th>
<tr><th colspan="2" style="text-align: left">Nucleotide Differences in Consensus Genomes (FreeBayes as reference)</th>
<tr><td colspan="2" style="padding-left: 20px">G5631N CT12453NN T25398N ATT27774NNN</td></tr>
</table></td>
<td style="vertical-align: top"><img src="coverage/benchmark-bad16_coverage_plot.png"></td>
</body>
</html>
