<!DOCTYPE html>
<html>
<head></head>
<body>
<h3>SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), version v1.5.6&nbsp;&nbsp;(<a href="https://github.com/jaleezyy/covid-19-signal">https://github.com/jaleezyy/covid-19-signal</a>)</h3>
<p>Note: Asterisks (*) indicates a discrepancy between iVar (default) and FreeBayes (if run)</p>
<h3>Sample: SC2285</h3>
<p><table><tr>
<td style="vertical-align: top"><table>
<tr><th colspan="2" style="text-align: left">Pangolin Lineage Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">B.1.36.18</td></tr>
<tr><th colspan="2" style="text-align: left">Nextclade Clade Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">20C</td></tr>
<tr><th colspan="2" style="text-align: left">Data Volume [ <a href="adapter_trimmed/SC2285_trim_galore.log">SC2285_trim_galore.log</a> ]</th>
<tr><td style="padding-left: 20px">Raw Data (read pairs)</td><td>213934</td></tr>
<tr><td style="padding-left: 20px">Raw Data (base pairs)</td><td>107394868</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (read pairs)</td><td>213934</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (base pairs)</td><td>96535865</td></tr>
<tr><td style="padding-left: 20px">Post Trim (read pairs)</td><td>212504</td></tr>
<tr><th colspan="2" style="text-align: left">Quality Control Flags</th>
<tr><td style="padding-left: 20px">Genome Fraction greater than 90%</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">No indels detected (maximum length 85bp)</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Depth of coverage >= 2000x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">All variants with at least 90% frequency among reads</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Frameshifts in SARS-CoV-2 open reading frames</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Reads per base sequence quality</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Sequencing adapter removed</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 100x</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 1000x</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">FASTQC Flags [ <a href="adapter_trimmed/SC2285_R1_val_1_fastqc.html">SC2285_R1_val_1_fastqc.html</a> | <a href="adapter_trimmed/SC2285_R2_val_2_fastqc.html">SC2285_R2_val_2_fastqc.html</a> ]</th>
<tr><td style="padding-left: 20px">Per base sequence content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per sequence GC content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per tile sequence quality</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Sequence Length Distribution</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">Kraken2 [ <a href="kraken2/SC2285_kraken2.report">SC2285_kraken2.report</a> ]</th>
<tr><td style="padding-left: 20px">Reads SARS-CoV-2 (%)</td><td>74.1</td></tr>
<tr><th colspan="2" style="text-align: left">QUAST [ <a href="quast/SC2285_quast_report.html">SC2285_quast_report.html</a> ]</th>
<tr><td style="padding-left: 20px">Genome Length (bp)</td><td>29904.0</td></tr>
<tr><td style="padding-left: 20px">Genome Fraction (%)</td><td>88.045</td></tr>
<tr><td style="padding-left: 20px">N's per 100 kbp</td><td>12225.79</td></tr>
<tr><td style="padding-left: 20px">Genomic Features</td><td>14 + 8 part</td></tr>
<tr><td style="padding-left: 20px">Mismatches</td><td>26.0</td></tr>
<tr><td style="padding-left: 20px">Mismatches per 100 kbp</td><td>98.75</td></tr>
<tr><td style="padding-left: 20px">Indels</td><td>3.0</td></tr>
<tr><td style="padding-left: 20px">Indels per 100 kbp</td><td>11.39</td></tr>
<tr><td style="padding-left: 20px">Average Depth of Coverage</td><td>2120.1</td></tr>
<tr><td style="padding-left: 40px">Fraction with 0 coverage</td><td>0.034</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1x-10x coverage</td><td>0.089</td></tr>
<tr><td style="padding-left: 40px">Fraction with 11x-100x coverage</td><td>0.222</td></tr>
<tr><td style="padding-left: 40px">Fraction with 101x-1000x coverage</td><td>0.305</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1001x-2000x coverage</td><td>0.133</td></tr>
<tr><td style="padding-left: 40px">Fraction with 2001x-10000x coverage</td><td>0.153</td></tr>
<tr><td style="padding-left: 40px">Fraction with > 10000x coverage</td><td>0.065</td></tr>
<tr><td style="padding-left: 20px">5' Ns</td><td>29</td></tr>
<tr><td style="padding-left: 20px">3' Ns</td><td>238</td></tr>
<tr><th colspan="2" style="text-align: left">Variants in Consensus Genome (iVar)</th>
<tr><td colspan="2" style="padding-left: 20px">T630-TC* C841A G1406T A1597-G* T1947C C2110T A2304-G* C3037T C3140T T3981C</td></tr>
<tr><td colspan="2" style="padding-left: 20px">A6731G G8027T A8968G C10044-TG* G10446T G11990+A* G12080A C12710G T13642-CTTA*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C14408T G14866T G15380T G15438T C15929T C16304T C18877T C18981T T19350-A*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C20812T A20829-AC* T21376-C* G21785C C21928T C22444T C22747T A23403G A24072-T*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C24563+T* C24566A* G24893C C24899+A* C25207T C25357A G25563T C25587-A* G26020T</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C26735T C28093T T28196C C28475-G* C28854T G29543T C29555T</td></tr>
<tr><th colspan="2" style="text-align: left">Unique Variants in Consensus Genome (FreeBayes)</th>
<tr><td colspan="2" style="padding-left: 20px">TCTTA13642T TA19350T A22904T AT24072A C24566TA CA25587C CG28475C</td></tr>
<tr><th colspan="2" style="text-align: left">Nucleotide Differences in Consensus Genomes (FreeBayes as reference)</th>
<tr><td colspan="2" style="padding-left: 20px">TC631NN G1598N G2305N N8320C TG10045NN T13642TNNNN T19346TN AC20825NN C21372N</td></tr>
<tr><td colspan="2" style="padding-left: 20px">W22899A A24561M C24894CN C25582CN N25667T C28469CN</td></tr>
</table></td>
<td style="vertical-align: top"><img src="coverage/SC2285_coverage_plot.png"></td>
</body>
</html>
