<!DOCTYPE html>
<html>
<head></head>
<body>
<h3>SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), version v1.5.6&nbsp;&nbsp;(<a href="https://github.com/jaleezyy/covid-19-signal">https://github.com/jaleezyy/covid-19-signal</a>)</h3>
<p>Note: Asterisks (*) indicates a discrepancy between iVar (default) and FreeBayes (if run)</p>
<h3>Sample: SC2197</h3>
<p><table><tr>
<td style="vertical-align: top"><table>
<tr><th colspan="2" style="text-align: left">Pangolin Lineage Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">B.1.564.1</td></tr>
<tr><th colspan="2" style="text-align: left">Nextclade Clade Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">20C</td></tr>
<tr><th colspan="2" style="text-align: left">Data Volume [ <a href="adapter_trimmed/SC2197_trim_galore.log">SC2197_trim_galore.log</a> ]</th>
<tr><td style="padding-left: 20px">Raw Data (read pairs)</td><td>228000</td></tr>
<tr><td style="padding-left: 20px">Raw Data (base pairs)</td><td>114456000</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (read pairs)</td><td>228000</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (base pairs)</td><td>106348670</td></tr>
<tr><td style="padding-left: 20px">Post Trim (read pairs)</td><td>226806</td></tr>
<tr><th colspan="2" style="text-align: left">Quality Control Flags</th>
<tr><td style="padding-left: 20px">Genome Fraction greater than 90%</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">No indels detected (maximum length 85bp)</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Depth of coverage >= 2000x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">All variants with at least 90% frequency among reads</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Frameshifts in SARS-CoV-2 open reading frames</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Reads per base sequence quality</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Sequencing adapter removed</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 100x</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 1000x</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">FASTQC Flags [ <a href="adapter_trimmed/SC2197_R1_val_1_fastqc.html">SC2197_R1_val_1_fastqc.html</a> | <a href="adapter_trimmed/SC2197_R2_val_2_fastqc.html">SC2197_R2_val_2_fastqc.html</a> ]</th>
<tr><td style="padding-left: 20px">Per tile sequence quality</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per base sequence content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per sequence GC content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Sequence Length Distribution</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">Kraken2 [ <a href="kraken2/SC2197_kraken2.report">SC2197_kraken2.report</a> ]</th>
<tr><td style="padding-left: 20px">Reads SARS-CoV-2 (%)</td><td>83.71</td></tr>
<tr><th colspan="2" style="text-align: left">QUAST [ <a href="quast/SC2197_quast_report.html">SC2197_quast_report.html</a> ]</th>
<tr><td style="padding-left: 20px">Genome Length (bp)</td><td>29902.0</td></tr>
<tr><td style="padding-left: 20px">Genome Fraction (%)</td><td>93.522</td></tr>
<tr><td style="padding-left: 20px">N's per 100 kbp</td><td>6745.37</td></tr>
<tr><td style="padding-left: 20px">Genomic Features</td><td>14 + 8 part</td></tr>
<tr><td style="padding-left: 20px">Mismatches</td><td>15.0</td></tr>
<tr><td style="padding-left: 20px">Mismatches per 100 kbp</td><td>53.64</td></tr>
<tr><td style="padding-left: 20px">Indels</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">Indels per 100 kbp</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">Average Depth of Coverage</td><td>2615.6</td></tr>
<tr><td style="padding-left: 40px">Fraction with 0 coverage</td><td>0.024</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1x-10x coverage</td><td>0.046</td></tr>
<tr><td style="padding-left: 40px">Fraction with 11x-100x coverage</td><td>0.202</td></tr>
<tr><td style="padding-left: 40px">Fraction with 101x-1000x coverage</td><td>0.356</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1001x-2000x coverage</td><td>0.115</td></tr>
<tr><td style="padding-left: 40px">Fraction with 2001x-10000x coverage</td><td>0.172</td></tr>
<tr><td style="padding-left: 40px">Fraction with > 10000x coverage</td><td>0.085</td></tr>
<tr><td style="padding-left: 20px">5' Ns</td><td>30</td></tr>
<tr><td style="padding-left: 20px">3' Ns</td><td>238</td></tr>
<tr><th colspan="2" style="text-align: left">Variants in Consensus Genome (iVar)</th>
<tr><td colspan="2" style="padding-left: 20px">C1059T C3037T C3326A T3402C C3773T T5322-AG* G5666T A5671C C6195T G6320T C7518T</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C8480T T10084C T12267G A12329-G* G13601-T* T14191C C14408T G14488T G15262T</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C16173T C16260T G16549T T16625C C17083T C18303T* G18653-CAC* A20465G G20560+T*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C20964-T* C21090T C21133-AA* A21137T* G23264T A23403G A24253T A24411-T* G25563T</td></tr>
<tr><td colspan="2" style="padding-left: 20px">T25673C T26609-AC* T26624C C26681T T26813-G* A27379T C28821A G28954-A*</td></tr>
<tr><th colspan="2" style="text-align: left">Unique Variants in Consensus Genome (FreeBayes)</th>
<tr><td colspan="2" style="padding-left: 20px">C15126T T15164C TTGGC18299CTGGT,TTGGT GCAC18653G</td></tr>
<tr><th colspan="2" style="text-align: left">Nucleotide Differences in Consensus Genomes (FreeBayes as reference)</th>
<tr><td colspan="2" style="padding-left: 20px">CNNNN321CTTT G4283K AG5323NN G12330N Y15126C Y15164T G18653GNNN T20962N</td></tr>
<tr><td colspan="2" style="padding-left: 20px">AA21131NN A21134W N22875A T24409N AC26607NN G26811N A28952N</td></tr>
</table></td>
<td style="vertical-align: top"><img src="coverage/SC2197_coverage_plot.png"></td>
</body>
</html>
