# Example SIGNAL Output

Standalone example output from the SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL, https://github.com/jaleezyy/covid-19-signal).

## Citation

Nasir et al. SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), a Snakemate workflow for rapid and bulk analysis of Illumina sequencing of SARS-CoV-2 genomes. Manuscript in preparation.

## Data

Throughout the COVID-19 pandemic, the SIGNAL team has contributed to routine surveillance within Ontario, Canada, in collaboration with Public Health Ontario, Sunnybrook Health Sciences Centre, Mount Sinai Hospital, Hamilton Health Sciences, and others as a member of the ONCoV Genomics Coalition. To demonstrate SIGNAL output and applicability, we re-examined a dataset of 113 samples from November 6-10, 2020 that underwent ARTIC3 PCR (https://github.com/artic-network/primer-schemes) and Illumina paired-end 250 bp sequencing using a HiSeq instrument. At the time of sequencing, 57 of 113 (50.4%) of this sequencing run passed all quality control metrics for submission of consensus genome sequences to the GISAID database.

All sequencing reads have been deposited in BioProject PRJNA689621. All consensus genome sequences and associated metadata in this dataset are published in GISAIDâ€™s EpiCoV database: to view the contributors of each individual sequence with details such as accession number, Virus name, Collection date, Originating Lab and Submitting Lab and the list of Authors, visit DOI:10.55876/gis8.240105tb.

## Software
* SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), version v1.5.6  
* [PANGOLIN versions](/results_dir/final_pangolin_versions.txt) 
* [NextClade versions](/results_dir/final_nextclade_versions.txt)

