<!DOCTYPE html>
<html>
<head></head>
<body>
<h3>SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), version v1.5.6&nbsp;&nbsp;(<a href="https://github.com/jaleezyy/covid-19-signal">https://github.com/jaleezyy/covid-19-signal</a>)</h3>
<p>Note: Asterisks (*) indicates a discrepancy between iVar (default) and FreeBayes (if run)</p>
<h3>Sample: SC2267</h3>
<p><table><tr>
<td style="vertical-align: top"><table>
<tr><th colspan="2" style="text-align: left">Pangolin Lineage Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">B.1.36.18</td></tr>
<tr><th colspan="2" style="text-align: left">Nextclade Clade Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">20C</td></tr>
<tr><th colspan="2" style="text-align: left">Data Volume [ <a href="adapter_trimmed/SC2267_trim_galore.log">SC2267_trim_galore.log</a> ]</th>
<tr><td style="padding-left: 20px">Raw Data (read pairs)</td><td>469816</td></tr>
<tr><td style="padding-left: 20px">Raw Data (base pairs)</td><td>235847632</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (read pairs)</td><td>469816</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (base pairs)</td><td>229497299</td></tr>
<tr><td style="padding-left: 20px">Post Trim (read pairs)</td><td>463689</td></tr>
<tr><th colspan="2" style="text-align: left">Quality Control Flags</th>
<tr><td style="padding-left: 20px">Genome Fraction greater than 90%</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">No indels detected (maximum length 85bp)</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Depth of coverage >= 2000x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">All variants with at least 90% frequency among reads</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Frameshifts in SARS-CoV-2 open reading frames</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Reads per base sequence quality</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Sequencing adapter removed</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 100x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 1000x</td><td>PASS</td></tr>
<tr><th colspan="2" style="text-align: left">FASTQC Flags [ <a href="adapter_trimmed/SC2267_R1_val_1_fastqc.html">SC2267_R1_val_1_fastqc.html</a> | <a href="adapter_trimmed/SC2267_R2_val_2_fastqc.html">SC2267_R2_val_2_fastqc.html</a> ]</th>
<tr><td style="padding-left: 20px">Per base sequence content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per sequence GC content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per tile sequence quality</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Sequence Length Distribution</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">Kraken2 [ <a href="kraken2/SC2267_kraken2.report">SC2267_kraken2.report</a> ]</th>
<tr><td style="padding-left: 20px">Reads SARS-CoV-2 (%)</td><td>93.02</td></tr>
<tr><th colspan="2" style="text-align: left">QUAST [ <a href="quast/SC2267_quast_report.html">SC2267_quast_report.html</a> ]</th>
<tr><td style="padding-left: 20px">Genome Length (bp)</td><td>29903.0</td></tr>
<tr><td style="padding-left: 20px">Genome Fraction (%)</td><td>99.763</td></tr>
<tr><td style="padding-left: 20px">N's per 100 kbp</td><td>237.43</td></tr>
<tr><td style="padding-left: 20px">Genomic Features</td><td>21 + 3 part</td></tr>
<tr><td style="padding-left: 20px">Mismatches</td><td>23.0</td></tr>
<tr><td style="padding-left: 20px">Mismatches per 100 kbp</td><td>77.1</td></tr>
<tr><td style="padding-left: 20px">Indels</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">Indels per 100 kbp</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">Average Depth of Coverage</td><td>5999.3</td></tr>
<tr><td style="padding-left: 40px">Fraction with 0 coverage</td><td>0.001</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1x-10x coverage</td><td>0.001</td></tr>
<tr><td style="padding-left: 40px">Fraction with 11x-100x coverage</td><td>0.001</td></tr>
<tr><td style="padding-left: 40px">Fraction with 101x-1000x coverage</td><td>0.069</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1001x-2000x coverage</td><td>0.09</td></tr>
<tr><td style="padding-left: 40px">Fraction with 2001x-10000x coverage</td><td>0.68</td></tr>
<tr><td style="padding-left: 40px">Fraction with > 10000x coverage</td><td>0.158</td></tr>
<tr><td style="padding-left: 20px">5' Ns</td><td>4</td></tr>
<tr><td style="padding-left: 20px">3' Ns</td><td>67</td></tr>
<tr><th colspan="2" style="text-align: left">Variants in Consensus Genome (iVar)</th>
<tr><td colspan="2" style="padding-left: 20px">C241T T1947C C2455T C3037T A6731G G6894T G8027T C14408T G15438T C17977T C18877T</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C18981T G21785C C21928T C22444T A23403G C25207T G25563T C26735T C28087T C28093T</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C28854T C29555T</td></tr>
<tr><th colspan="2" style="text-align: left">Unique Variants in Consensus Genome (FreeBayes)</th>
<tr><th colspan="2" style="text-align: left">Nucleotide Differences in Consensus Genomes (FreeBayes as reference)</th>
</table></td>
<td style="vertical-align: top"><img src="coverage/SC2267_coverage_plot.png"></td>
</body>
</html>
