<!DOCTYPE html>
<html>
<head></head>
<body>
<h3>SARS-CoV-2 Illumina GeNome Assembly Line (SIGNAL), version v1.6.6&nbsp;&nbsp;(<a href="https://github.com/jaleezyy/covid-19-signal">https://github.com/jaleezyy/covid-19-signal</a>)</h3>
<p>Note: Asterisks (*) indicates a discrepancy between iVar (default) and FreeBayes (if run)</p>
<h3>Sample: hCoV-19-Wales-PHWC-4C8F5E-2021</h3>
<p><table><tr>
<td style="vertical-align: top"><table>
<tr><th colspan="2" style="text-align: left">Pangolin Lineage Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">B.1.1.7</td></tr>
<tr><th colspan="2" style="text-align: left">Nextclade Clade Assignment</th>
<tr><td colspan="2" style="padding-left: 20px">20I</td></tr>
<tr><th colspan="2" style="text-align: left">Data Volume [ <a href="adapter_trimmed/hCoV-19-Wales-PHWC-4C8F5E-2021_trim_galore.log">hCoV-19-Wales-PHWC-4C8F5E-2021_trim_galore.log</a> ]</th>
<tr><td style="padding-left: 20px">Raw Data (read pairs)</td><td>154538</td></tr>
<tr><td style="padding-left: 20px">Raw Data (base pairs)</td><td>42902700</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (read pairs)</td><td>154538</td></tr>
<tr><td style="padding-left: 20px">Post Primer Removal (base pairs)</td><td>42702821</td></tr>
<tr><td style="padding-left: 20px">Post Trim (read pairs)</td><td>154537</td></tr>
<tr><th colspan="2" style="text-align: left">Quality Control Flags</th>
<tr><td style="padding-left: 20px">Genome Fraction greater than 90%</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">No indels detected (maximum length 85bp)</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Depth of coverage >= 2000x</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">All variants with at least 90% frequency among reads</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Frameshifts in SARS-CoV-2 open reading frames</td><td>WARN</td></tr>
<tr><td style="padding-left: 20px">Reads per base sequence quality</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">Sequencing adapter removed</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 100x</td><td>PASS</td></tr>
<tr><td style="padding-left: 20px">At least 90% of positions have coverage >= 1000x</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">FASTQC Flags [ <a href="adapter_trimmed/hCoV-19-Wales-PHWC-4C8F5E-2021_R1_val_1_fastqc.html">hCoV-19-Wales-PHWC-4C8F5E-2021_R1_val_1_fastqc.html</a> | <a href="adapter_trimmed/hCoV-19-Wales-PHWC-4C8F5E-2021_R2_val_2_fastqc.html">hCoV-19-Wales-PHWC-4C8F5E-2021_R2_val_2_fastqc.html</a> ]</th>
<tr><td style="padding-left: 20px">Per base sequence content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Per sequence GC content</td><td>FAIL</td></tr>
<tr><td style="padding-left: 20px">Sequence Length Distribution</td><td>WARN</td></tr>
<tr><th colspan="2" style="text-align: left">Kraken2 [ <a href="kraken2/hCoV-19-Wales-PHWC-4C8F5E-2021_kraken2.report">hCoV-19-Wales-PHWC-4C8F5E-2021_kraken2.report</a> ]</th>
<tr><td style="padding-left: 20px">Reads SARS-CoV-2 (%)</td><td>95.22</td></tr>
<tr><th colspan="2" style="text-align: left">QUAST [ <a href="quast/hCoV-19-Wales-PHWC-4C8F5E-2021_quast_report.html">hCoV-19-Wales-PHWC-4C8F5E-2021_quast_report.html</a> ]</th>
<tr><td style="padding-left: 20px">Genome Length (bp)</td><td>29884.0</td></tr>
<tr><td style="padding-left: 20px">Genome Fraction (%)</td><td>99.027</td></tr>
<tr><td style="padding-left: 20px">N's per 100 kbp</td><td>1315.08</td></tr>
<tr><td style="padding-left: 20px">Genomic Features</td><td>19 + 5 part</td></tr>
<tr><td style="padding-left: 20px">Mismatches</td><td>35.0</td></tr>
<tr><td style="padding-left: 20px">Mismatches per 100 kbp</td><td>118.2</td></tr>
<tr><td style="padding-left: 20px">Indels</td><td>4.0</td></tr>
<tr><td style="padding-left: 20px">Indels per 100 kbp</td><td>13.51</td></tr>
<tr><td style="padding-left: 20px">Average Depth of Coverage</td><td>1275.5</td></tr>
<tr><td style="padding-left: 40px">Fraction with 0 coverage</td><td>0.003</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1x-10x coverage</td><td>0.014</td></tr>
<tr><td style="padding-left: 40px">Fraction with 11x-100x coverage</td><td>0.037</td></tr>
<tr><td style="padding-left: 40px">Fraction with 101x-1000x coverage</td><td>0.51</td></tr>
<tr><td style="padding-left: 40px">Fraction with 1001x-2000x coverage</td><td>0.212</td></tr>
<tr><td style="padding-left: 40px">Fraction with 2001x-10000x coverage</td><td>0.224</td></tr>
<tr><td style="padding-left: 40px">Fraction with > 10000x coverage</td><td>0.0</td></tr>
<tr><td style="padding-left: 20px">5' Ns</td><td>54</td></tr>
<tr><td style="padding-left: 20px">3' Ns</td><td>67</td></tr>
<tr><th colspan="2" style="text-align: left">Variants in Consensus Genome (iVar)</th>
<tr><td colspan="2" style="padding-left: 20px">C241T C913T C3037T C3267T C5388A C5986T T6954C T8603C G11287-TCTGGTTTT* T11288A*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">C12076T C14408T C14676T C15279T T16176C A17615G G19398T A21764-TACATG*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">T21990-TTA* T22213G A23063T C23271A A23403G C23604A C23709T T24506G G24914C</td></tr>
<tr><td colspan="2" style="padding-left: 20px">G25437T C26082T C27972T G28048T A28111G T28270-A* G28280C* A28281T* T28282A*</td></tr>
<tr><td colspan="2" style="padding-left: 20px">G28881A* G28882A* G28883C* C28977T</td></tr>
<tr><th colspan="2" style="text-align: left">Unique Variants in Consensus Genome (FreeBayes)</th>
<tr><td colspan="2" style="padding-left: 20px">GTCTGGTTTT11287G ATACATG21764A TTTA21990T TA28270T GAT28280CTA GGG28881AAC</td></tr>
<tr><th colspan="2" style="text-align: left">Nucleotide Differences in Consensus Genomes (FreeBayes as reference)</th>
</table></td>
<td style="vertical-align: top"><img src="coverage/hCoV-19-Wales-PHWC-4C8F5E-2021_coverage_plot.png"></td>
</body>
</html>
